### ISSUE TITLE
automating token generation in python 

### ISSUE THREAD

**Issue by priyakumariengineer** [GireeshS22/upstox#8]
Hi @GireeshS22 

do you have any code for automatic token generation in python  or suggesgtion , if yes can you add that code in repository ??

**Comment by GireeshS22** [comment]
No. I manually generate token and put it in the program each session

**Comment by priyakumariengineer** [comment]
is there any way , like headless browser or something we can automate token generation ??
